siliconcompiler/report/streamlit_viewer.py

```diff
@@ -477,16 +458,9 @@ def display_flowgraph_toggle(label_after):
         label_after (bool) : the default label fro the toggle
     """
     # this horizontally aligns the toggle with the header
-    streamlit.markdown("\n")
-    # TODO: By October, streamlit will have their own toggle widget
-    fg_toggle = st_toggle_switch(label=" ",
-                                 key="flowgraph_toggle",
-                                 default_value=label_after,
-                                 label_after=True,
-                                 # the colors are optional
-                                 inactive_color=INACTIVE_TOGGLE_COLOR,
-                                 active_color=ACTIVE_TOGGLE_COLOR,
-                                 track_color=TRACK_TOGGLE_COLOR)
+    streamlit.markdown("")
+    streamlit.markdown("")
+    fg_toggle = not streamlit.checkbox('Hide')
```

Comment:
When this is hidden it still says Hide, but it's not clear what it is doing. This should probably be changed to indicate this is the flowgraph somehow.